.github/workflows/miniconda-win.yml

```diff
@@ -0,0 +1,38 @@
+name: windows
+
+on:
+  pull_request:
+  push:
+    branches: [master]
+
+jobs:
+  run:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        python-version: [3.7, 3.9]
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup Conda
+      uses: s-weigand/setup-conda@v1
```

Comment:
Any particular reason you've opted for this one and not [conda-icubator/setup-miniconda](https://github.com/conda-incubator/setup-miniconda)?

Reply:
It is less bloated and does less, just activate the pre-installed conda for you. Which is faster. I also had multiple issues with all the yaml options in the other one. I'd rather have the same commands on the config as those that I type on a terminal to have a 1-1 comparison if I need to debug. (With that said, I do break that rule here adding the conda-forge channel with a yaml config option :smile:)

Reply:
Makes sense.